src/components/FiatValue.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <span
-    v-if="fiatValue"
```

Comment:
Won't it show `~ null` in the case [if `this.rate` is falsy](https://github.com/aeternity/superhero-ui/blob/e7975882f7cf87748520e7d7a84ebf3ba2137d47/src/components/FiatValue.vue#L35)?